resources/js/views/Locations/SortLocations.vue

```diff
@@ -1,5 +1,12 @@
 <template>
     <section class="locations-main" :class="container">
+
+        <div class="columns py-4 px-4">
+            <div class="column is-half is-offset-3">
+                <Download type="worldwide"/>
```

Comment:
Small change, but call this "global" pls

Reply:
Ok, this is actually not used, it's there only because the Download component requires a type.